.circleci/config.yml

```diff
@@ -16,8 +16,17 @@ jobs:
     executor: macos-executor
     steps:
       - checkout
-      - run: brew install physfs sdl2 sdl2_image sdl2_mixer sdl2_ttf glew
+      - restore_cache:
+          key: deps-v2-NAS2D-{{ arch }}
+      - run: HOMEBREW_NO_AUTO_UPDATE=1 brew install physfs sdl2 sdl2_image sdl2_mixer sdl2_ttf glew
+      - run: brew link physfs sdl2 sdl2_image sdl2_mixer sdl2_ttf glew
       - run: make -k
+      - save_cache:
```

Comment:
I think the `save_cache` should be moved directly after the `brew` commands. The `make -k` should be done after the cache save.

Reply:
I don't remember why I put it in the restore/save block in the first place. It probably can be moved since it doesn't touch or modify the brew download folder or the brew install folder.